enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -179,23 +193,23 @@ def can_redeem(self, learner_id, content_key):
         return False
 
 
-class CappedEnrollmentLearnerCreditAccessPolicy(LearnerCreditAccessPolicy):
+class CappedEnrollmentLearnerCreditAccessPolicy(SubsidyAccessPolicy):
     """
     Policy that limits the maximum amount that can be spent aggregated across all users covered by this policy
     """
-    spend_limit = models.IntegerField(
-        blank=True,
-        default=0,
-    )
+
+    objects = PolicyManager()
```

Comment:
should we name this after the class?

Reply:
This manager will be used by all the different sub-classes. Should we name it after parent class?